src/BirdRecordSecondary.java

```diff
@@ -0,0 +1,58 @@
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Abstract class BirdRecordSecondary.
+ */
+public abstract class BirdRecordSecondary implements BirdRecord {
+
+    List<Bird> BIRDS;
```

Comment:
This shouldn't be here. It should be accessible through the kernel methods only.